src/Namotion.Reflection.Tests/XmlDocsExtensionsTests.cs

```diff
@@ -229,7 +332,201 @@ public void When_summary_has_paramref_tag_then_it_is_converted()
             var summary = typeof(WithParamrefTagInXmlDoc).GetMethod("Foo").GetXmlDocsSummary();
 
             //// Assert
-            Assert.Equal("Returns name.", summary);
+            Assert.Equal("Returns param1 followed by param2.", summary);
+        }
```

Comment:
Updated existing test to adapt to slight refactoring for `WithParamrefTagInXmlDoc.Foo()` method which now takes 2 parameters instead of one.
https://github.com/RicoSuter/Namotion.Reflection/pull/171/files#r2062572845